pkg/odo/cli/component/push.go

```diff
@@ -42,59 +40,53 @@ const PushRecommendedCommandName = "push"
 
 // PushOptions encapsulates options that push command uses
 type PushOptions struct {
-	ignores          []string
+	ignores []string
+	show    bool
+
 	sourceType       config.SrcType
 	sourcePath       string
-	localConfig      *config.LocalConfigInfo
 	componentContext string
+	client           *occlient.Client
```

Comment:
I htink this can be safely removed
bcoz context would have a client

Reply:
We need this for `isPrjExists, err := project.Exists(po.client, prjName)` so it has to remain here..